guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc

```diff
@@ -0,0 +1,224 @@
+[id="upgrading-{project-context}-or-proxy-in-place-using-leapp_{context}"]
+= Upgrading {Project} or {SmartProxy} to {EL} 9 In-Place Using Leapp
+
+Use this procedure to upgrade your {Project} or {SmartProxy} installation from {EL} 8 to {EL} 9.
+
+.Prerequisites
+* {Project} {ProjectVersion} or {SmartProxy} {ProjectVersion} running on {EL} 8.
+ifdef::satellite[]
+* Review Known Issues before you begin an upgrade.
+For more information, see {ReleaseNotesDocURL}ref_known-issues_assembly_introducing-red-hat-satellite[Known Issues in {ProjectName} {ProjectVersion}].
+endif::[]
+ifndef::satellite[]
+* Access to available repositories or a local mirror of repositories.
+endif::[]
+
+ifeval::["{mode}" == "disconnected"]
+.Prerequisites for Disconnected Environment
+If you run {Project} in a disconnected environment, ensure it also meets the following prerequisites:
+
+* You require access to {RHEL} and {Project} packages.
+Obtain the ISO files for {RHEL} 9 and {Project}.
+For more information, see {UpgradingDisconnectedDocURL}Upgrading_satellite_upgrading-disconnected[Upgrading Red Hat Satellite] in _{UpgradingDisconnectedDocTitle}_.
+* For more information on customizing the Leapp upgrade for your environment, see https://access.redhat.com/articles/4977891[Customizing your {RHEL} in-place upgrade].
+* Since Leapp completes part of the upgrade in a container that has no access to additional ISO mounts, the repositories cannot be served from a locally mounted ISO but must be delivered over the network from a different machine.
+* For more information, see https://access.redhat.com/solutions/7030156[How to in-place upgrade an offline / disconnected RHEL 8 machine to RHEL 9 with Leapp?]
+endif::[]
+
+ifdef::satellite[]
+[NOTE]
+====
+* {Project} supports DEFAULT and FIPS crypto-policies.
+The FUTURE crypto-policy is not supported for {Project} and {SmartProxy} installations.
+====
+endif::[]
```

Comment:
I need to figure out if this note is still relevant

Reply:
To elaborate a bit more: The note is technically still correct, we do not support FUTURE policy.
However, I do not think there is a way to change the policy while doing an in-place upgrade, so I don't see a good reason for the note to be here.

Reply:
@ehelms @ekohl any objections? This is IMHO relevant for fresh installations and general "system requirements", but not here.

I double-checked with Leapp and they do not allow to change the policy during the upgrade, so if the system is on supported policy on EL8 it will be on EL9 too.

Reply:
Sounds correct to me.

Reply:
So:
```suggestion
```